### ISSUE TITLE
Advanced Search page a11y test failures

### ISSUE THREAD

**Issue by PaulMorris** [SolutionGuidance/psm#687]
@cecilia-donnelly [reported](https://github.com/SolutionGuidance/psm/pull/676#issuecomment-368163476) that the a11y test for the provider "Advanced Search" page was failing with "this form has no submit button".  @PaulMorris was not able to reproduce this.  The test is `@ignore`d temporarily until we can troubleshoot this further.

**Comment by kfogel** [comment]
What does the line "Issue #518 Implement Accessibility Checking in CI" at the bottom of the description mean?  Is it saying that this issue (#687) is related to that one?

**Comment by PaulMorris** [comment]
@kfogel Yes that's the idea, although, looking at some other issues, it seems this convention we've been using is more for PRs and commits than issues.  (I've just been doing more commits and PRs than issues so got carried away.)  I think it'd be fine to remove it.  It's not adding much.
